fwupdmgr: add page (#6601)
tldr-pages/tldr: pages/linux/fwupdmgr.md

# fwupdmgr

> > A simple daemon that allows updating some device firmwares, including UEFI.

Reviewer: As far as I know, `fwupdmgr` is not the daemon, but the tool for communicating with it.
Does "firmware" have a plural?
I'd phrase it like this:
```suggestion
> A tool for updating device firmware, including UEFI, using `fwupd`.
```
